homeassistant/components/tuya/switch.py

```diff
@@ -296,6 +296,12 @@
         SwitchEntityDescription(
             key=DPCode.SWITCH,
             name="Switch",
+            icon="mdi:power",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.START,
+            name="Start",
```

Comment:
How does this switch work? Can you abort the start by turning it off again?
Or is the switch used to trigger an event (and you can't turn off the start after that).

Reply:
Can you abort the start by turning it off again?
Yes, you can turn OFF this switch and it stops cooking. I have tested the functionality with actual device.

The existing key=DPCode.SWITCH is a power button, but it does not start cooking. In order to start and stop cooking process, key=DPCode.START is needed.

Reply:
> Yes, you can turn OFF this switch and it stops cooking. 

Awesome! The reason I ask is that Tuya also uses booleans to trigger events (buttons in the HA world, not switches). But if you can turn it off, the switch is correct :)

Thanks for the clarification 👍 

../Frenck